lib/services/app-scaffolding-extensions-service.ts

```diff
@@ -22,9 +25,25 @@ export class AppScaffoldingExtensionsService extends dependencyExtensionsService
 	public prepareAppScaffolding(): IFuture<void> {
 		return (() => {
 			var appScaffoldingConfig = this.$dependencyConfigService.getAppScaffoldingConfig().wait();
-			var afterPrepareAction = () => this.$childProcess.exec("npm install", {cwd: this.appScaffoldingPath });
+			var afterPrepareAction = () => {
+				return (() => {
+					this.npmInstall("glob-watcher@0.0.8").wait(); // HACK: With this we are able to make paths shorter with 20 symbols.
+					this.npmInstall().wait();
+					this.npmDedupe().wait();
+				}).future<void>()();
+			};
 			this.prepareDependencyExtension(AppScaffoldingExtensionsService.APP_SCAFFOLDING_NAME, appScaffoldingConfig, afterPrepareAction).wait();
 		}).future<void>()();
 	}
+
+	protected npmInstall(packageToInstall?: string): IFuture<void> {
+		packageToInstall = packageToInstall || "";
+		var command = util.format("npm install %s %s", packageToInstall, "--production");
```

Comment:
The second %s is unneeded:

`npm install ${packageToInstall} --production` ftw!